codeforboston/maple
# Migrating Leaf components to Bootstrap/typescript, deleting CSS modules

# Summary

This PR migrates the Leaf components to Bootstrap/Typescript for #1458 

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.

# Screenshots

Identical to how leaves look on production.

<img width="616" alt="Screenshot 2024-02-29 at 12 29 06 AM" src="https://github.com/codeforboston/maple/assets/2694761/e24659b0-5aaa-4c24-beee-38007605141e">


# Known issues

N/A

# Steps to test/reproduce

1. Go to the home page
1. Scroll down and see the pretty leaves
